source/postprocess/gravity_point_values.cc

```diff
@@ -539,10 +552,10 @@ namespace aspect
       {
         prm.enter_subsection ("Gravity calculation");
         {
-          if (prm.get ("Sampling scheme") == "map")
-            sampling_scheme = map;
-          else if (prm.get ("Sampling scheme") == "list")
-            sampling_scheme = list;
+          if (prm.get ("Sampling scheme") == "uniform distribution")
+            sampling_scheme = uniform_distribution;
+          else if (prm.get ("Sampling scheme") == "list of points")
+            sampling_scheme = list_of_points;
```

Comment:
I agree with @tjhei, it is easy to add the old names here as two additional conditions, which will keep backward compatibility without complicating the code. Just use the new names for `sampling_scheme`, but allow old input files to work.